client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -11,11 +11,14 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import Button from 'components/forms/form-button';
 import AuthorMapping from './author-mapping-item';
 import SiteUsersFetcher from 'components/site-users-fetcher';
 import UsersStore from 'lib/users/store';
 
+import ActionButtonContainer from 'my-sites/importer/importer-action-buttons/container';
+import ActionButton from 'my-sites/importer/importer-action-buttons/action-button';
+import CloseButton from 'my-sites/importer/importer-action-buttons/close-button';
+
```

Comment:
These components seem odd to me, in that they have generic names but are importing specific. Could we have more descriptive names, like `ImportActionContainer`, `ImportActionButton`, `ImportCancelButton`, etc?

(I know this issue isn't new to this PR)